0044 026 (#1932)
vegaprotocol/specs: protocol/0044-LIME-lp_mechanics.md

- For a futures market with `market.liquidity.earlyExitPenalty = 0.25` and `total stake = target stake + 140` already, if the following transactions occur:

  - `LP1` places a transaction to reduce their stake by `30`

>   - `LP2`  places a transaction to reduce their stake by `100`, and then wait till end of the epoch so the engine can pick up the reduction of their stake,

Reviewer: ```suggestion
  - `LP2`  places a transaction to reduce their stake by `100`, and then wait until end of the epoch,
```
